java/pom.xml

```diff
@@ -414,7 +414,7 @@
             greater than 3.1.2 (tested up to 3.6.0) when compiling
             arrow-tools about Jackson being only used for tests.
           -->
-          <version>3.1.2</version>
+          <version>3.6.1</version>
```

Comment:
Is the comment above this line still applicable?

Reply:
Need to make a choice regarding the option we should take.